packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts

```diff
@@ -23,28 +26,31 @@ export class CreateExampleProject {
   constructor(readonly cliOptions: CliOptions) {
     this.baseDirPath = process.cwd();
 
-    this.exampleFolderName = this.getExampleFolderName();
-    this.targetExampleFolderPath = path.join(
+    this.targetDirectoryName = this.getTargetFolderName();
+    this.targetDirectoryPath = path.join(
       this.baseDirPath,
-      this.exampleFolderName,
+      this.targetDirectoryName,
     );
 
     this.askUIControllerUrl = 'https://docs.askui.com/docs/general/Components/AskUI-Controller';
     this.helperTemplateConfig = {};
   }
 
-  private getExampleFolderName(): string {
-    if (this.cliOptions.projectName) {
-      return this.cliOptions.projectName.trim();
+  private getTargetFolderName(): string {
+    if (this.cliOptions.askuiWorkflows) {
+      if (/\s/g.test(this.cliOptions.askuiWorkflows.trim())) {
+        throw new TargetDirectoryError('Directory name must not contain whitespaces');
```

Comment:
Minor (optional) things:
- Would not use a special error (name) like `TargetDirectoryError` here. For me, this is a classic `ValueError` or `ValidationError` (also custom errors in Node.js, some use the buildin `TypeError` or `AssertionError`, last one together with `assert`, for these kinds of errors) where some input (here a cli option) is not as expected. Would rather use such a kind of error as this is, then, clearer what kind of error we are dealing with.
- Normally, you would validate the cli options as soon as they are passed where possible and not mix it with the logic that builds upon them (⟶ Fail fast, separation of concerns etc.). This is why some frameworks for building clis already include validators.